terraform/modules/prometheus/input.tf

```diff
@@ -12,20 +14,21 @@ variable disk_quota { default = 1024 }
 
 variable influxdb_service_instance_id {}
 
-variable alert_rules {
-  default = ""
-}
+variable alert_rules { default = "" }
 
-variable config_file {
-  default = ""
-}
+variable scrape_file { default = "" }
```

Comment:
This is not a filename but a string. Could we rename to something like `extra_scrape_config`?

Reply:
Changed